specification/sql/resource-manager/Microsoft.Sql/preview/2020-02-02-preview/DevOpsAudit.json

```diff
@@ -0,0 +1,326 @@
+        "parameters": [
+          {
+            "$ref": "../../../common/v1/types.json#/parameters/ResourceGroupParameter"
+          },
+          {
+            "$ref": "#/parameters/ServerNameParameter"
+          },
+          {
+            "name": "devOpsAuditPolicyName",
+            "in": "path",
+            "description": "The name of the devops audit policy. This should always be 'default'.",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "name": "parameters",
+            "in": "body",
+            "description": "Properties of DevOps audit policy",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/ServerDevOpsAuditingPolicy"
+            }
+          },
+          {
+            "$ref": "../../../common/v1/types.json#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "../../../common/v1/types.json#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Successfully updated the DevOps audit settings.",
+            "schema": {
+              "$ref": "#/definitions/ServerDevOpsAuditingPolicy"
+            }
+          },
+          "default": {
+            "description": "*** Error Responses: ***\n\n * 400 InvalidServerDevOpsAuditPolicyCreateRequest - The create server DevOps audit policy request does not exist or has no properties object.\n\n * 400 DataSecurityInvalidUserSuppliedParameter - An invalid parameter value was provided by the client.\n\n * 400 DevOpsAuditInvalidStorageAccountCredentials - The provided storage account or access key is not valid.\n\n * 404 SubscriptionDoesNotHaveServer - The requested server was not found\n\n * 404 ServerNotInSubscriptionResourceGroup - Specified server does not exist in the specified resource group and subscription.\n\n * 409 ServerDevOpsAuditPolicyInProgress - Set server DevOps audit is already in progress."
+          },
+          "202": {
+            "description": "Updating the audit DevOps settings is in progress."
+          }
+        },
+        "x-ms-long-running-operation": true,
+        "x-ms-long-running-operation-option": "Azure-AsyncOperation",
+        "x-ms-examples": {
+          "Update a server's DevOps audit policy with minimal input": {
+            "$ref": "./examples/ServerDevOpsAuditCreateMin.json"
+          },
+          "Update a server's DevOps audit policy with all params": {
+            "$ref": "./examples/ServerDevOpsAuditCreateMax.json"
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Sql/servers/{serverName}/devOpsAuditingSettings": {
+      "get": {
+        "tags": [
+          "DevOpsAudit"
+        ],
+        "description": "Lists DevOps audit settings of a server.",
+        "operationId": "ServerDevOpsAuditPolicies_ListByServer",
```

Comment:
The operation id group name does not match either this file name (DevOpsAudit) nor the entity name (devOpsAuditingSettings). What's the reason for this discrepency?

Reply:
there is no specific reason.
i will change the swagger name from DevOpsAudit to ServerDevOpsAudit